apps/backend/src/main/resources/api.yaml

```diff
@@ -1,22 +1,985 @@
 openapi: "3.0.0"
 
 info:
-  title: "storeme cloud API"
+  title: "StoreMe API"
   version: "1"
 
 servers:
   - url: http://localhost:8091
-    description: catalogue api
+    description: Main API
+
+tags:
+  - name: Item
+    description: Relates to items
+  - name: ItemModel
+    description: Relates to item models
+  - name: Location
+    description: Relates to locations
+  - name: Movement
+    description: Relates to movements
+  - name: User
+    description: Relates to users
+  - name: Inventory
+    description: Relates to inventories
+  - name: Jwt
+    description: Relates to JWT
 
 paths:
-  /example:
+  /authentication/getToken:
+    post:
+      tags:
+        - Jwt
+      summary: Get token
+      description: Get a JWT token for a username and password.
+      operationId: getToken
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CredentialsDTO'
+      responses:
+        '200':
+          description: Operation successful.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/JWTDTO'
+        400:
+          $ref: '#/components/responses/400'
+        401:
+          $ref: '#/components/responses/401'
+        403:
+          $ref: '#/components/responses/403'
+        404:
+          $ref: '#/components/responses/404'
+        409:
+          $ref: '#/components/responses/409'
+        500:
+          $ref: '#/components/responses/500'
+  /authentication/validateToken:
+    post:
+      tags:
+        - Jwt
+      summary: Validate token
+      description: Verify that a token can be used.
+      operationId: validateToken
+      parameters:
+        - name: Authorization
+          in: header
+          schema:
+            type: string
+      responses:
+        '200':
+          description: If the JWT is valid.
+        400:
+          $ref: '#/components/responses/400'
+        401:
+          $ref: '#/components/responses/401'
+        403:
+          $ref: '#/components/responses/403'
+        404:
+          $ref: '#/components/responses/404'
+        409:
+          $ref: '#/components/responses/409'
+        500:
+          $ref: '#/components/responses/500'
+      security:
+        - JWTAuth: [ ]
+  /users:
+    get:
+      tags:
+        - User
+      description: Get users, supports the swagger-undocumented filter
+      operationId: getUsers
+      responses:
+        200:
+          description: OK
+          content:
+            'application/json':
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/UserDTO'
```

Comment:
Where is returned the pagination information (at least `total`)?

It can be returned in a header, if it is easier for you.

Reply:
I think you've been mistaken, or I did not understand your question, but there is no pagination whatsoever

Reply:
> there is no pagination whatsoever.

It should have one (at least `total`).
Especially for a large number of entries.

Example:
Given the (hypothetical) URL: `/movements?where[locationId]=1&[order][createdAt]=desc&limit=100&skip=100`
It should return something like:
```json
{
  "data": ["..."],
  "total": 15463248
}
```

> **Note**:
> I usually name `pagination` all extra information that helps creating a `pagination` system.
> `total` being the more important, and information, such as `range-start` or `range-end`, are optional.
> If it is simply a problem of terminology.